template/azurePipelines/kustomize/.pipelines/azure-kubernetes-service.yaml

```diff
@@ -5,8 +5,8 @@ variables:
   armServiceConnection: {{ARMSERVICECONNECTION}}
   azureContainerRegistry: {{AZURECONTAINERREGISTRY}}
   containerName: {{CONTAINERNAME}}
-  acrRg: {{CLUSTERRESOURCEGROUP}}
-  clusterRg: {{ACRRESOURCEGROUP}}
+  acrRg: {{ACRRESOURCEGROUP}}

```

Comment:
💡Minor questions: Looks good but minor reference: Do we have any doc around this env vars for this tool level thing?

Reply:
No, we don't have any doc yet. I'll add the descriptions of each var in the template in a separate PR